### ISSUE TITLE
zenith reference in documentation / code.

### ISSUE THREAD

**Issue by phoenix24** [neondatabase/neon#1702]
While reviewing code, I noticed a lot of references to `zenith` in [docs](https://github.com/neondatabase/neon/blob/main/docs/glossary.md)/ [rfcs](https://github.com/neondatabase/neon/blob/main/docs/multitenancy.md) and code.

I do not have the historical context here but looks like the project was renamed in the recent past, and these are remaining fragments - should these be renamed with `neon`?

I'm happy to raise pull-requests towards it, but wanted to confirm the assumption.

**Comment by SomeoneToIgnore** [comment]
Yes, `zenith` is the old name and indeed some rename is ongoing.
Not sure what to propose here generally, some things might break more due to renames, but we'll see it in the CI.

**Comment by yeputons** [comment]
We also have many names like `ZId`, `ZTenantTimelineId`, `ZNodeId`, and even `zenith_fixtures` in our test framework. It would be good to rename at some point as well.

**Comment by phoenix24** [comment]
@yeputons thank you for pointing that out, I'm picking this up now. Would the corresponding change be good - 
| old name | new name |
|--------|--------|
| `ZId` | `NId` |
| `ZTenantTimelineId` | `NTenantTimelineId` |
| `.zenith` | `.neon` |
| `zenith_fixtures` | `neon_fixtures` | 

@hlinnaka thoughts?

**Comment by SomeoneToIgnore** [comment]
I suggest to drop the `Z`/`N` whatever suffixes from the `pub struct` API part, so just have `TenantTimelineId`, `TenantId`, etc.

**Comment by phoenix24** [comment]
thank you @SomeoneToIgnore! sounds good to me, on it 👍
